app/utils/logger.js

```diff
@@ -59,6 +59,9 @@ const nsDebugLogger = createLogger({
 const nsDebug = process.env.NS_DEBUG === 'true'
 
 module.exports = (loggerName) => {
+  if (process.env.GOVUK_PAY__USE_BASIC_LOGGER === 'true') {
```

Comment:
you can achieve a simple logger without adding another env var by just toggling `NS_DEBUG` (which probably shouldn't have been merged 🫠 ). 

If that's still too much detail you can comment out the `metadataStr` in `nsDebugFormat` above to have just single log lines (with the added benefit of colour for info, warn, error etc)

Reply:
I just tried that, the log level display is nice but I'm getting a whole load of extra lines specifying the `logger_name`:

```
2024-12-17 12:02:12 [info]: Page viewed with account simplification turned on
{
  "logger_name": "/Users/natalie.carey/projects/alphagov/pay-selfservice/app/models/User.class.js"
}
```

Given the choice I would still go with the *very* basic option, but I could work with your approach.

I think we should standardise to one or the other and I'd suggest the `GOVUK_PAY__USE_BASIC_LOGGER` name because people can set that globally or project-by-project without risking conflicts and understanding what they're getting.

Reply:
yeah see the second part of my comment re `metadataStr` if you want it to look like the above screenshot